### PR TITLE
Fix issue with overviewText + misc.

### PR DESCRIPTION
**Pull request by Jawshy**
#### Changes proposed in this pull request: 
- Remove `overviewText` from mission.sqm files and add it to description.ext.
  - The `overviewText` will be displayed below the `overviewPicture` during loading.
  - Known issue: `overviewText` will not display on the role selection
screen/mission lobby (description box empty in top left). 
  - Resolves the following message from arma3server.rpt: 

```
Altis Life RPG v5.0.0: string @STR_MISC_overviewText cannot be localized client-side - move to global stringtable
```

- Wrapped stringtable keys in quotes for consistency. Although, it does not seem to make a difference either way. 
- Tweaked comment for `disableChannels`.

### CODE REVIEW

File: Altis_Life.Altis/description.ext [hunk 1]
```diff
@@ -1,4 +1,6 @@
-disableChannels[]={{0,true,true},{1,true,true},{2,true,true}};    // Disabled text and voice for global, side, and command channels. disableChannels[] = {{channelID<number>, disableChat<bool>, disableVoice<bool>}};
+disableChannels[] = {{0,true,true},{1,true,true},{2,true,true}};    // Disabled text and voice for global, side, and command channels. 
+                                                                    // Syntax: disableChannels[] = {{channelID<number>, disableChat<bool>, disableVoice<bool>}};
+overviewText = "$STR_MISC_overviewText";    // Text to be displayed below the overviewPicture on the mission selection screen when the mission is available to play.
```
Comment (BoGuu) [inline Altis_Life.Altis/description.ext]: Multi line comments? Looks a bit spaghetti

Comment (BoGuu) [inline Altis_Life.Altis/description.ext]: Multi line comment? Looks spaghetti ;)

Comment (Jawshy) [inline Altis_Life.Altis/description.ext]: @BoGuu, [Config_Master.hpp, lines 25-26](https://github.com/AsYetUntitled/Framework/blob/4cf67cf12a61055803c3f60b07b51c65f184ea26/Altis_Life.Altis/config/Config_Master.hpp#L25-L26).

**Comment by Jawshy** [comment]
LGTM

**Comment by BoGuu** [comment]
LGTM

**Pull request merged**